src/main/java/mate/academy/spring/controller/MovieController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.dto.MovieRequestDto;
+import mate.academy.spring.model.dto.MovieResponseDto;
+import mate.academy.spring.service.MovieService;
+import mate.academy.spring.service.mapper.MovieDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    private final MovieService movieService;
+    private final MovieDtoMapper movieDtoMapper;
+
+    public MovieController(MovieService movieService, MovieDtoMapper movieDtoMapper) {
+        this.movieService = movieService;
+        this.movieDtoMapper = movieDtoMapper;
+    }
+
+    @GetMapping("/")
```

Comment:
```suggestion
    @GetMapping
```
Fix it everywhere below, please)